packages/rules-unit-testing/src/api/index.ts

```diff
@@ -15,15 +15,16 @@
  * limitations under the License.
  */
 
-import * as firebase from 'firebase';
+import firebase from 'firebase';
 import { _FirebaseApp } from '@firebase/app-types/private';
 import { FirebaseAuthInternal } from '@firebase/auth-interop-types';
 import * as request from 'request';
 import { base64 } from '@firebase/util';
 import { setLogLevel, LogLevel } from '@firebase/logger';
 import { Component, ComponentType } from '@firebase/component';
 
-export { database, firestore } from 'firebase';
+const { firestore, database } = firebase;
+export { firestore, database };
```

Comment:
@samtstern @yuchenshi 
Meant to ask you before merging the PR - How does developer use these two exports? Do they use the typings from them?
They won't get typings after this change because `firestore` and `database` are not namespaces anymore.

Reply:
I believe they're used to provide access to Firestore / RTDB classes like this:
https://github.com/firebase/firebase-js-sdk/blob/master/packages/rules-unit-testing/test/database.test.ts#L313

If those tests still pass I think we're fine?  To be honest I don't quite understand namespaces.

Reply:
Yeah, that will continue to work. We are good. Thanks for confirming.